src/app/tests/suites/TestFabricSyncBridgedNode.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test Fabric Synchronization condition on Bridged Node Device Type
+
+config:
+    nodeId: 0x12344321
+
+tests:
+    - label:
+          "Read the DeviceTypeList attribute of the Descriptor cluster and
+          check whether the device type is Aggregator."
+      endpoint: 1
```

Comment:
If you put the endpoint in the test step, it's always going to run this step against that endpoint.

Suggest adding endpoint to the config at the top with the CI default. That can be overridden in the TH / on the command line. Then remove from the individual steps.

Reply:
Done